src/geode/mesh/builder/solid_mesh_builder.cpp

```diff
@@ -305,46 +277,65 @@ namespace
         const geode::PolyhedronVertex& polyhedron_vertex,
         geode::index_t new_vertex_id )
     {
-        const auto facets_vertices =
```

Comment:
Why do you revert this change?

Reply:
Because the two versions are quasi similar in terms of Ir. I prefer the less verbose version.